strix/utilities/utils.py

```diff
@@ -81,6 +81,23 @@ def parse_datalist(filelist, format="auto", has_unlabel: bool = False):
         unlabel: str = DatalistKeywords.UNLABEL.value
         if label not in datalist:
             raise GenericException(f"Your datalist does not contain '{label}' key.")
+        elif "tensorImageSize" in datalist:
+            training_list = datalist["training"]
+            num = len(training_list)
+            dataset = []
+            for i in range(num):
+                case_data = {}
+                if "image" in training_list[i]:
+                    image_relpath = training_list[i]["image"]
+                    case_data['image'] = os.path.abspath(image_relpath)
```

Comment:
The abspath function will add the current path of workspace and the input path together. It should not be used here.